jsonpickle/handlers.py

```diff
@@ -196,11 +197,30 @@ def restore(self, data):
         return cls.__new__(cls, *params)
 
 
-DatetimeHandler.handles(datetime.datetime)
 DatetimeHandler.handles(datetime.date)
 DatetimeHandler.handles(datetime.time)
 
 
+class DatetimeObjectHandler(BaseHandler):
+    """Serialization for datetime.datetime objects"""
```

Comment:
This docstring could use some explanation as to why it exists instead of the generic reduce handlers or the handlers above (e.g. "Provide a user-readable and portable format (ISO-9660) for the underlying date value instead of the default reduction that stores an internal binary representation".